completions/bash/podman

```diff
@@ -1963,6 +1963,7 @@ _podman_container_run() {
 		boolean_options="$boolean_options
 			--detach -d
 			--rm
+                        --rmi
```

Comment:
spacing looks suspicious here.  Do we have a tab where there should be a space or vice versa?